manageiq-ui-classic.gemspec

```
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.executables   = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency "rails", ">= 5.0.0.1", "< 5.2"
+  s.add_dependency "rails", "< 5.3"
```

@juliancheal hakiri is committing a harakiri here because your setting allows to install any old version of rails.
What do you think about something like this:
```suggestion
  s.add_dependency "rails", ">= 5.0.0.1", "< 5.3"
```

---

Thanks @skateman